src/status_im/integration_test.cljs

```diff
@@ -231,5 +232,59 @@
        (logout!) (rf-test/wait-for [::logout/logout-method] ; we need to logout to make sure the node is not in an inconsistent state between tests
                                    (assert-logout))))))))
 
+(deftest delete-chat-test
+  (log/info "========= delete-chat-test ==================")
+  (rf-test/run-test-async
+   (initialize-app!)
+   (rf-test/wait-for
+    [:status-im.init.core/initialize-view]
+    (generate-and-derive-addresses!)
+    (rf-test/wait-for
+     [:multiaccount-generate-and-derive-addresses-success] ; wait for the keys
+     (assert-multiaccount-loaded)
+     (create-multiaccount!)
+     (rf-test/wait-for
+      [::transport/messenger-started]
+      (assert-messenger-started)
+      (rf/dispatch-sync [:chat.ui/start-chat chat-id]) ;; start a new chat
+      (rf-test/wait-for
+       [:status-im.chat.models/one-to-one-chat-created]
+       (rf/dispatch-sync [:chat.ui/navigate-to-chat chat-id])
+       (is (= chat-id @(rf/subscribe [:chats/current-chat-id])))
+       (is @(rf/subscribe [:chats/chat chat-id]))
+       (rf/dispatch-sync [:chat.ui/remove-chat-pressed chat-id])
+       (rf/dispatch-sync [:chat.ui/remove-chat chat-id])
+       (is (not @(rf/subscribe [:chats/chat chat-id])))
+       (is (nil? @(rf/subscribe [:chats/current-chat-id])))
+       (logout!) (rf-test/wait-for [::logout/logout-method] ; we need to logout to make sure the node is not in an inconsistent state between tests
+                                   (assert-logout))))))))
+
+(deftest mute-chat-test
+  (log/info "========= mute-chat-test ==================")
+  (rf-test/run-test-async
+   (initialize-app!)
+   (rf-test/wait-for
+    [:status-im.init.core/initialize-view]
+    (generate-and-derive-addresses!)
+    (rf-test/wait-for
+     [:multiaccount-generate-and-derive-addresses-success] ; wait for the keys
+     (assert-multiaccount-loaded)
+     (create-multiaccount!)
+     (rf-test/wait-for
+      [::transport/messenger-started]
+      (assert-messenger-started)
+      (rf/dispatch-sync [:chat.ui/start-chat chat-id]) ;; start a new chat
```

Comment:
[Code style] A single semicolon is preferred.

Reply:
Sorry, didn't post the reference: https://github.com/bbatsov/clojure-style-guide#margin-inline-comments

Reply:
![](https://i.ytimg.com/vi/CdnkmqhjRbs/maxresdefault.jpg)

Reply:
Haha, exactly 😎